src/options/optionsMain.ts

```diff
@@ -60,6 +60,8 @@ export async function getOptions(rawOptions: any): Promise<AppOptions> {
       disableDevTools: rawOptions.disableDevTools,
       disableGpu: rawOptions.disableGpu || false,
       diskCacheSize: rawOptions.diskCacheSize || null,
+      disableOldBuildWarning:
+        rawOptions.disableOldBuildWarningyesiknowitisinsecure || false,
```

Comment:
Hm, the `rawOptions` bag is generated by `commander`, which transforms `--my-option` into `rawOptions.myOption`. Thus, are you sure it's not

```suggestion
        rawOptions.disableOldBuildWarningYesiknowitisinsecure || false,
```